example/src/screens/InitialScreen.tsx

```diff
@@ -1,21 +1,42 @@
 import { BackgroundAnimation } from '@components/BackgroundAnimation';
 import { Typo } from '@components/Typo';
 import { CardButton } from '@components/buttons/CardButton';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import * as Sentry from '@sentry/react-native';
 import * as Application from 'expo-application';
 import React from 'react';
-import { View, StyleSheet, Image } from 'react-native';
+import { View, StyleSheet, Image, Pressable, Alert } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import { useVideoroomState } from 'src/VideoroomContext';
 
 export const InitialScreen = ({ navigation }) => {
+  const { setIsDevMode, isDevMode } = useVideoroomState();
+
+  const toggleDevMode = async () => {
+    const updatedIsDevMode = !isDevMode;
+    Alert.alert(
+      'Dev mode is now '.concat(updatedIsDevMode ? 'enabled' : 'disabled')
+    );
+    setIsDevMode(updatedIsDevMode);
+    try {
+      const jsonValue = JSON.stringify(updatedIsDevMode);
+      await AsyncStorage.setItem('isDevMode', jsonValue);
```

Comment:
Put it in setIdDevMode function in the context.The context is already managing reading the value, so it should also manage writing the value so it's all in one place.

Reply:
I did some magic there, since async store is... async it cannot be simply used in the useEffect. Wdyt about my current approach?

Reply:
Sg!